readme.adoc

```diff
@@ -88,16 +89,18 @@ The World Wide Web is a virtual network of information.
 == Usage
 
 These algorithms are exposed as Neo4j procedures.
-We can call them directly from Cypher in your Neo4j Browser, from cypher-shell, or your client code.
+We can call them directly from Cypher in your Neo4j Browser, from cypher-shell, or from your client code.
 
 For most algorithms we provide two procedures:
 
-* one named `algo.<name>` that writes results back to the graph as node-properties and reports statistics.
-* another named `algo.<name>.stream` that returns a stream of data, e.g. node-ids and computed values.
+* `algo.<name>` - this procedure writes results back to the graph as node-properties, and reports statistics.
+* `algo.<name>.stream` - this procedure returns a stream of data.
```

Comment:
you probably want to end this line in a `+` to have the next line attached to this bullet point.